pkg/network/driver/common.go

```diff
@@ -164,6 +166,16 @@ func (h *NetworkUtilsHandler) ConfigureIpForwarding(proto iptables.Protocol) err
 	return err
 }
 
+func (h *NetworkUtilsHandler) ConfigurePingGroupRange() error {
+	err := sysctl.New().SetSysctl(sysctl.PingGroupRange, "0 2147483647")
+	return err
+}
+
+func (h *NetworkUtilsHandler) ConfigureFsFileMax() error {
+	err := sysctl.New().SetSysctl(sysctl.FsFileMax, "9223372036854775807")
```

Comment:
whoa... why is the default setting not good enough for us?

Reply:
Added explanation to the PR message.
I removed this setting from virt-handler, so the cluster-admin will be responsible to set it on the nodes if he wants to use passt VMs.
It felt wrong to set a global sysctl property on the node from virt-handler just in case a passt PR ever ran on this node.
What do you think?

Reply:
> It felt wrong to set a global sysctl property on the node from virt-handler just in case a passt PR ever ran on this node.
What do you think?

well, it certainly raises questions. Does passt actually open an fd for every possible port right away, or is each fd opened on demand as ports are in use?

Essentially i'm trying to understand how likely it is to hit this file max limitation when default values are used. If passt always opens 2^18 ports for every VM, then it's likely we'll need to do something here, or at least come up with a way to allow people to opt into setting higher file max values in our KubeVirt CR.

However, if the 2^18 ports for every VM is theoretical based on actual traffic, then I'm not very concerned about actually surpassing the default values.

Reply:
It is not based on actual traffic. Every VM that is forwarding *all* ports opens ~2^18 sockets on startup.
I thought for the tech preview just documenting it as a requirement for the node.

Reply:
ouch. i think each FD involve some memory overhead, 2^18 isn't a trivial amount of memory.

Reply:
Yes, to run passt VM with no specific ports `vmi.Spec.Domain.Resources.Requests.Memory` should be at least `2048M`. Added it to the PR description as well.